src/obu.rs

```diff
@@ -2537,7 +2536,7 @@ unsafe fn parse_obus(c: &mut Rav1dContext, r#in: &Rav1dData, global: bool) -> Ra
                     &mut (*c).out.p,
                     &c.content_light,
                     &c.mastering_display,
-                    &c.itut_t35,
+                    &*c.itut_t35.read().unwrap(),
```

Comment:
For these `.read()`s of `itut_t35`, I'm making the `&` into `.clone()`s, which I think will fix this issue?  I'll put up a PR for it and let me know if it helps after rebasing.  It was one small commit in a larger change I'm working on, but I didn't realize it could be helpful elsewhere.

Reply:
I don't think that removes the need to lock the field, we immediately take it out which requires mutability.